Documentation/writing-principles-and-guidelines/spelling-and-typo3-terminology.rst

```diff
@@ -1,111 +1,1006 @@
+
+        The Admin Panel is an administrative tool that can be enabled in the
+        frontend to show debugging information, performed SQL queries and more
+        for authenticated backend users.
+
+    ..  _admin-tools:
+    -   *Admin Tools*
+
+        Admin tools are a group of backend modules. These include maintaining
+        the installation, adjusting settings, executing upgrade wizards,
+        checking environment information and setting up extensions.
+
+    ..  _allow-fields:
+    -   *Allow Fields*
+
+        Allow fields refer to fields of content elements displayed in the TYPO3
+        backend in regard to their permissions. Editors can only edit fields in
+        the backend, which are included in the list of "Allow fields" in their
+        permission setup.
+
+    ..  _assets:
+    -   *Assets*
+
+        Assets are media resources like images, videos and documents that are
+        uploaded and managed in the TYPO3 system. Also, extensions can include
+        assets which can be referred to in the frontend, like specific icons or
+        even JavaScript libraries.
+
+*   B
+
+    ..  _backend---frontend:
+    -   *Backend / Frontend*
+
+        Backend and Frontend are the two main areas of TYPO3 CMS. The backend is
+        the administrative interface for editors and administrators, while the
+        frontend is the publicly accessible part of the website.
+
+    ..  _backend-bookmarks:
+    -   *Backend Bookmarks*
+
+        Backend bookmarks are shortcuts that users can set for frequently used
+        backend pages for quicker access.
+
+    ..  _backend-layout:
+    -   *Backend Layout*
+
+        The backend layout defines the structure and design of the backend user
+        interface in regard to maintaining content elements and the arrangement
+        of their input fields. A backend layout can be set on a page-level, and
+        this attribute can actually be also evaluated in the frontend, to affect
+        the arrangement of elements on a page.
+
+    ..  _backend-module:
+    -   *Backend Module*
+
+        Backend modules are extendable components in the TYPO3 backend that
+        provide various functionalities and tools like user management or file
+        management. The left hand panel in the backend shows all these modules.
+
+*   C
+
+    ..  _callout:
+    -   *Callout*
+
+        A callout is a highlighted element designed to draw attention to
+        important information or actions.
+
+    ..  _cache--cache-backend--frontend-cache:
+    -   *Cache (Cache Backend, Frontend Cache)*
+
+        Caches are used to improve website performance by storing frequently
+        accessed data. The backend cache stores data in the administration
+        system, while the frontend cache stores data for the website display.
+
+    ..  _cache-tags:
+    -   *Cache Tags*
+
+        Cache tags are labels assigned to cache entries to facilitate the
+        management and invalidation of cached data.
```

Comment:
Labels is a very short description of the contained data. But "PageId_243-2" is not.

What do you think about:

```suggestion
        With cache tags one or more cache entries can be grouped together. That way 
        all cache entries related to a cache tag can be invalidated with just one call.
```